src/utils/sformat.fs

```diff
@@ -916,9 +918,9 @@ namespace Microsoft.FSharp.Text.StructuredFormat
             match c with 
```

Comment:
I'm not sure but perhaps we should be escaping some entire classes of Unicode characters as well. See https://en.wikipedia.org/wiki/Template:General_Category_(Unicode).  Though I suppose it's not strictly necessary - the rule is that F# should be able to parse what it prints.